freqtrade/rpc/telegram.py

```diff
@@ -216,7 +216,7 @@ def send_msg(self, msg: Dict[str, Any]) -> None:
 
             msg['emoji'] = self._get_sell_emoji(msg)
 
-            message = ("{emoji} *{exchange}:* Selling {pair}\n"
+            message = ("{emoji} *{exchange}:* Selling {pair} (#{trade_id})\n"
```

Comment:
i would think that we should add this to the `cancel*` branches too - otherwise you'd end up with a cancelled order but can't tie that to the trade.

Reply:
Good point, I'll add it